oracle/service.go

```diff
@@ -315,9 +376,26 @@ func (s *oracleSvc) composeProviderFeedMsgs(priceBatch []*PriceData) (result []c
 	return result
 }
 
+func (s *oracleSvc) composeStorkOracleMsgs(priceBatch []*PriceData) (result []cosmtypes.Msg) {
+	if len(priceBatch) == 0 {
+		return nil
+	}
+
+	for _, priceData := range priceBatch {
+		msg := &oracletypes.MsgRelayStorkPrices{
```

Comment:
the batch includes all the price data structs from all providers, It needs to check the provider type, the batch
```suggestion
		if priceData.OracleType != oracletypes.OracleType_Stork {
			continue
		}
		msg := &oracletypes.MsgRelayStorkPrices{
```